packages/@o3r/eslint-config/src/rules/utils.cjs

```diff
@@ -0,0 +1,20 @@
+/**
+ * Convert the rules set as `warn` into `error`
+ * @param {import('@typescript-eslint/utils').TSESLint.FlatConfig.Config | import('@typescript-eslint/utils').TSESLint.FlatConfig.ConfigArray} config config to process
```

Comment:
not clear?

Reply:
The 1st one is for the param name 
The 2nd one is the 1st word of the description

Reply:
ah yes sorry it wasn't clear, I will resolve this